hack/lib/util.sh

```diff
@@ -32,7 +32,7 @@ kube::util::wait_for_url() {
   local i
   for i in $(seq 1 "$times"); do
     local out
-    if out=$(curl --max-time 1 -gkfs "$url" 2>/dev/null); then
+    if out=$(curl --max-time 5 -gkfs "$url" 2>/dev/null); then
```

Comment:
Right now, we try this by default 30 times, with 1 second apart. Total time to run before timeout would be 1 minute. Changing this to 5 means that it takes 6 seconds per retry, or 3 minutes to time out completely. Is 2 a more reasonable number here?

Reply:
Ack @cblecker i'll ask the folks running the tests which combination works best ...

cc @kiwik @animationzl

Reply:
Hi @cblecker ,

In our testing,

Reply:
@liu-sheng Does it still work if you lower 5 down to 2? I'd be more comfortable with that, then all the way up to 5.

Reply:
@cblecker I have tried with 2, seems it also has the failure possibility, but I have recognized that I used a low performance VM to testing. would you like to expose an environment variable to make that configurable ?

Reply:
@liu-sheng good point, let me do that